plugins/foreman_virt_who_configure/0.5/index.md

```diff
@@ -0,0 +1,272 @@
+    Exclude host parents: 
+    Debug mode:           no
+    Ignore proxy:
+```
+
+#### Creating a config
+
+```sh
+hammer virt-who-config create --name="gCBHwsVPFZ" --debug="1" --interval="60" --hypervisor-id="hostname" --hypervisor-type="esx" --organization-id="1" --filtering-mode="none" --satellite-url="foreman.example.com" --hypervisor-server="esxi.example.com" --hypervisor-username="admin" --hypervisor-password="password"
+
+Virt Who configuration [gCBHwsVPFZ] created
+```
+
+#### Updating a config
+
+```sh
+$ hammer virt-who-config update --id 5 --hypervisor-password="changeme"
+
+Virt Who configuration [gCBHwsVPFZ] updated
+```
+
+#### Deleting a config
+
+```sh
+$ hammer virt-who-config delete --id 5
+
+Virt Who configuration deleted
+```
+
+## 3.3 Monitoring
+
+Once the virt-who configuration is successfully deployed, the virt-who daemon runs and periodically send reports about what VMs it found on the hypervisors. Every report upload updates the configuration in Foreman, so you can monitor the configuration status. If you navigate to virt-who configurations list page, one of the table columns is called __Status__. Each configuration can have one of the following status:  
+
+* **No Report Yet** - This means, no virt-who report was recieved by Foreman yet, either the configuration was not deployed yet or there was some error during the deployment or virt-who can't connect to Foreman
+* **OK** - We received the report within the scheduled interval, everything is OK
+* **No Change** - virt-who does not send any report when it does not find any change on hypervisor, if we don't receive any report during the scheduled interval, we consider the configuration as unchanged. This could also happen if virt-who fails to upload reports for whatever reason, if you know there was new VM added on the hypervisor but you see the configuration in __No Change__ state, Check that virt-who is running and its logs.
+ 
+There's also a dasboard widget that you can watch. It contains a table with a number of configurations per each state. If you click on the status name, you'll see all configurations in this state. The widget also lists last three configurations in __No Change__ state as this might or might not be problematic.
+ 
+On the configurations lists page you can also search based on the state. The search syntax can be used
+
+```
+status = unknown
+status = out_of_date
+status = ok
+status = ok or status = out_of_date
+```
+
+## 3.4 Redeploying
+
+If there's some change you need to do to existing virt-who configuration, simply edit it via the same form that was used during creation. To edit a configuration, click __Edit__ button in the configuration table on a particular row. After you submit the new values, the configuration script reflects it. You can simply run the new version of the script and it will reconfigure the virt-who according to the new values. It also restarts the service for you. You can run the script repeatedly.   
```

Comment:
```suggestion
If there's some change you need to do to existing virt-who configuration, simply edit it via the same form that was used during creation. To edit a configuration, click the __Edit__ button in the configuration table on a particular row. After you submit the new values, the configuration script reflects it. You can simply run the new version of the script and it will reconfigure the virt-who according to the new values. It also restarts the service for you. You can run the script repeatedly.   
```